homeassistant/components/shelly/sensor.py

```diff
@@ -404,12 +464,12 @@ def native_value(self) -> StateType:
 
         return self.last_state
 
-    @property
-    def state_class(self) -> str | None:
-        """State class of sensor."""
-        return self.description.state_class
+    async def async_added_to_hass(self) -> None:
```

Comment:
Moved out of `ShellySleepingBlockAttributeEntity` as `state_class` doesn't apply outside sensor platform